README.rst

```diff
@@ -18,3 +18,10 @@ Prerequisites
 * `Traits <https://pypi.python.org/pypi/traits>`_
 * `NumPy <https://pypi.python.org/pypi/numpy>`_
 * `SciPy <https://pypi.python.org/pypi/scipy>`_
+
+Development Environment Setup
```

Comment:
Thank you for these changes!

In the interests of avoiding feature creep, and getting PRs merged effectively, please could you move these changes to a separate PR? Then we can keep this one focused on the issues it was originally trying to solve.

I have a few comments on the README updates, but I'll save them for the new PR.